All EIPIP Meetings/Meeting 028.md

```diff
@@ -0,0 +1,551 @@
+# EIP# EIP IP Meeting 28 Notes
+## Meeting Date/Time: Wednesday 10 March at 15:00 UTC
+### Meeting Duration: 1 hour
+### [GitHub Agenda Page](https://github.com/ethereum-cat-herders/EIPIP/issues/56)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=fKQVb-s0Tzs)
+### Moderator: Pooja Ranjan
+### Notes: Jim Bennett
+
+----
+## DECISION ITEMS
+
+**DECISION 28.1**: Continue to think about how to deal with suggested changes in Last Call and keep Item #2 on the agenda for the next meeting.[see 20:23](https://youtu.be/fKQVb-s0Tzs?t=1219)
+
+**DECISION 28.2**: Put an item on the next meeting agenda about EIPs for Berlin moving to Last Call. [see 32:39](https://youtu.be/fKQVb-s0Tzs?t=1959)
+
+**DECISION 28.3**: Discuss separating ERCs from the EIP repo in the next call. [see 39:53](https://youtu.be/fKQVb-s0Tzs?t=1959)
+----
+**Pooja Ranjan**
+Welcome to EIPIP meeting 28. I'm Pooja Ranjan.
+
+## AGENDA
+
+## 1. [Progress on EIP GitHub repo action bot](https://youtu.be/fKQVb-s0Tzs?t=4)
+The first item on the agenda is progress on EIP GitHub repo action bot. So a little more from the ECH team is working on the EIP GitHub repo. And I suppose she also created a pull requests to share this bot for review with the EIP editors. The pull request number is 3351. If there is any other update on the progress and concern with the bot that anyone would like to share in development of this effort, feel free to share, because I don't see any data here. But I have seen some comments left by the editors on the issue.
```

Comment:
```suggestion
The first item on the agenda is progress on EIP GitHub repo action bot. So, Alita Moore from the ECH team is working on the EIP GitHub repo. And I suppose she also created a pull requests to share this bot for review with the EIP editors. The pull request number is 3351. If there is any other update on the progress and concern with the bot that anyone would like to share in development of this effort, feel free to share, because I don't see Alita here. But I have seen some comments left by the editors on the issue.
```